### PR TITLE
:truck: update: Add 125th CatchUp summary

### PR DESCRIPTION
**Pull request by SirusCodes**

### CODE REVIEW

File: summary/sessions/125/content.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    ** Twitter open-sourced it's algorithm on GitHub at link:https://github.com/twitter/the-algorithm[twitter/the-algorithm^]
+* link:https://twitter.com/harshgkapadia[Harsh Kapadia^] and link:https://twitter.com/rishit_dagli[Rishit Dagli^] shared a few funny PR's from link:https://github.com/twitter/the-algorithm[twitter/the-algorithm^] repository.
+    ** A PR which removed link:https://github.com/twitter/the-algorithm/issues/236[author_is_elon^] which priorities Elon Musk's tweets.
+    ** link:https://github.com/twitter/the-algorithm/pull/774[Humra change merge karo!^]
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^], and link:https://twitter.com/kaku_jay[Jay Kaku^] cleared our confusion between what link:https://en.wikipedia.org/wiki/Bit_numbering[Most Significant Bit and Least Significant^], link:https://stackoverflow.com/a/19063231/13268307[Higher Order Bit and Lower Order Bit^], and link:https://www.section.io/engineering-education/what-is-little-endian-and-big-endian/[Little Endian and Big Endian^].
```
Comment (PranavDani) [inline summary/sessions/125/content.adoc:14]:
```suggestion
* link:https://twitter.com/rishit_dagli[Rishit Dagli^], and link:https://twitter.com/kaku_jay[Jay Kaku^] cleared our confusion between what link:https://en.wikipedia.org/wiki/Bit_numbering[Most Significant Bit and Least Significant^], link:https://stackoverflow.com/a/19063231/13268307[Higher Order Bit and Lower Order Bit^], and link:https://www.section.io/engineering-education/what-is-little-endian-and-big-endian/[Little Endian and Big Endian^].
```
```suggestion
* link:https://twitter.com/rishit_dagli[Rishit Dagli^], and link:https://twitter.com/kaku_jay[Jay Kaku^] cleared our confusion between link:https://en.wikipedia.org/wiki/Bit_numbering[Most Significant Bit and Least Significant Bit^], link:https://stackoverflow.com/a/19063231/13268307[Higher Order Bit and Lower Order Bit^], and link:https://www.section.io/engineering-education/what-is-little-endian-and-big-endian/[Little Endian and Big Endian^] is.
```

File: summary/sessions/125/content.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* link:https://twitter.com/kaku_jay[Jay Kaku^] shared link:https://youtu.be/iCI3B5eT9NA[Paul Horowitz's Interview^] who is the author of link:https://www.amazon.in/Art-Electronics-Paul-Horowitz/dp/0521809266[The Art of Electronics^] which is a considered as the bible for people working with electronics.
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] shared slides of link:https://docs.google.com/presentation/d/1eRUOqLul90ipgj-Uku-pswtSrwdZHXj9QAin-NNQKMc/edit#slide=id.g4a66e23a62_2_343[best practices for making figures].
+* link:https://twitter.com/SirusTweets[Darshan Rander^] talked about link:http://wasmio.tech/[WASM IO 2023].
```
Comment (PranavDani) [inline summary/sessions/125/content.adoc:17]:
```suggestion
* link:https://twitter.com/SirusTweets[Darshan Rander^] talked about link:http://wasmio.tech[WASM IO 2023^].
```

File: summary/sessions/125/content.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    ** How it is a new upcoming field and so many companies are building tools around it.
```
Comment (PranavDani) [inline summary/sessions/125/content.adoc:18]:
```suggestion
    ** How is it a new upcoming field and so many companies are building tools around it?
```

File: summary/sessions/125/content.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    ** Garbage collector in WASM is near and already available for beta users of Chrome users.
```
Comment (PranavDani) [inline summary/sessions/125/content.adoc:19]:
```suggestion
    ** Garbage collector in WASM is near and already available for beta Chrome users.
```

Comment (PranavDani) [reply summary/sessions/125/content.adoc:19]: WASM is near .... (launch date?)

File: summary/sessions/125/content.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+* link:https://twitter.com/kaku_jay[Jay Kaku^] shared about a new kind of cab booking service link:https://nammayatri.in/[Namma Yatri^] which is based on link:https://en.wikipedia.org/wiki/Open_Network_for_Digital_Commerce[ONDC's Beckn Protocol^].
```
Comment (PranavDani) [inline summary/sessions/125/content.adoc:20]:
```suggestion
* link:https://twitter.com/kaku_jay[Jay Kaku^] shared a new kind of cab booking service link:https://nammayatri.in[Namma Yatri^] which is based on link:https://en.wikipedia.org/wiki/Open_Network_for_Digital_Commerce[ONDC's Beckn Protocol^].
```

File: summary/sessions/125/content.adoc [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+==== Project Showcased
+
+* link:https://twitter.com/___LUNAGOK___[Kalpesh Jangir^] showcased link:https://github.com/LUNAGOK23/Attend-Assist[Attendance Assist] which is an app that allows user to mark their attendance with a to-do list feature for assignment.
```
Comment (PranavDani) [inline summary/sessions/125/content.adoc:24]:
```suggestion
* link:https://twitter.com/___LUNAGOK___[Kalpesh Jangir^] showcased _Attendance Assist_, an app allowing users to mark their attendance with a to-do list feature for assignments.
       ** link:https://github.com/LUNAGOK23/Attend-Assist[GitHub Repository^]
```

File: summary/sessions/125/content.adoc [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+* link:https://twitter.com/JiaHarisinghani[Jia Harisinghani^] showcased an HTML website where she is trying to learn Web Dev basics.
+* link:https://twitter.com/furtado_jaden[Jaden Furtado^] showcased his final year project, which is building a tool that checks for security vulnerabilities in your code.
```
Comment (PranavDani) [inline summary/sessions/125/content.adoc:26]:
```suggestion
* link:https://twitter.com/furtado_jaden[Jaden Furtado^] showcased his final year project which is a tool that checks for security vulnerabilities in code.
```

File: summary/sessions/125/content.adoc [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+    ** He has also built it in a way where someone can deploy it to their CI/CD pipeline and get a vulnerability report.
```
Comment (PranavDani) [inline summary/sessions/125/content.adoc:27]:
```suggestion
    ** He built it in a way such that someone can deploy it to their CI/CD pipeline and get a vulnerability report.
```

**Review by PranavDani — CHANGES_REQUESTED** [review]

File: summary/sessions/125/content.adoc [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    ** link:https://github.com/twitter/the-algorithm/pull/774[Humra change merge karo!^]
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^], and link:https://twitter.com/kaku_jay[Jay Kaku^] cleared our confusion between what link:https://en.wikipedia.org/wiki/Bit_numbering[Most Significant Bit and Least Significant^], link:https://stackoverflow.com/a/19063231/13268307[Higher Order Bit and Lower Order Bit^], and link:https://www.section.io/engineering-education/what-is-little-endian-and-big-endian/[Little Endian and Big Endian^].
+* link:https://twitter.com/kaku_jay[Jay Kaku^] shared link:https://youtu.be/iCI3B5eT9NA[Paul Horowitz's Interview^] who is the author of link:https://www.amazon.in/Art-Electronics-Paul-Horowitz/dp/0521809266[The Art of Electronics^] which is a considered as the bible for people working with electronics.
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] shared slides of link:https://docs.google.com/presentation/d/1eRUOqLul90ipgj-Uku-pswtSrwdZHXj9QAin-NNQKMc/edit#slide=id.g4a66e23a62_2_343[best practices for making figures].
```
[the remark this answers is not in the document]
Comment (SirusCodes) [reply summary/sessions/125/content.adoc:16]: best practices guideline? Seems a bit off to me

File: summary/sessions/125/content.adoc (hunk 3 above)
Comment (SirusCodes) [reply summary/sessions/125/content.adoc:18]: It an upcoming field... I don't know how to give context to that...

Comment (SirusCodes) [reply summary/sessions/125/content.adoc:18]: Maybe I phrased is wrong.

Comment (SirusCodes) [reply summary/sessions/125/content.adoc:18]:
```suggestion
    ** It is a new field, and many companies are building tools around it.
```

File: summary/sessions/125/content.adoc (hunk 4 above)
Comment (SirusCodes) [reply summary/sessions/125/content.adoc:19]: There is no launch date... Maybe Google will push a stable version this IO

File: summary/sessions/125/content.adoc (hunk 7 above)
Comment (SirusCodes) [reply summary/sessions/125/content.adoc:26]: Did that during the catchup itself

**Comment by SirusCodes** [comment]
Thanks a lot @PranavDani for being kind and writing suggestions🥺

**Pull request merged**